### PR TITLE
Make email clickable

### LINKED ISSUE #852: Make the email address clickable in help(about section) page
The email address is not a link as of now

### PR DESCRIPTION
**Pull request by cdhiraj40**
Fixes #852 

https://user-images.githubusercontent.com/75211982/140390457-08a01b95-9350-4089-a857-00cd48d83e67.mp4

### CODE REVIEW

**Comment by asaikarthikeya** [comment]
Hello @cdhiraj40 , Please test the app on the develop branch. The above issue doesnot exist in the develop branch, we soon plan to update the master with these changes present in develop. Thanks ;) resolved in #788

**Comment by cdhiraj40** [comment]
hey @asaikarthikeya, thats different part of help section as i can see from your recording too from #788

File: org.envirocar.app/res/layout/activity_help_layout.xml [hunk 1]
```diff
@@ -41,8 +41,8 @@
 
         <TextView
             style="@style/enviroCar.Help.content"
-            android:text="@string/help_content_1"
-            android:autoLink="web"/>
+            android:autoLink="email|web"
```
Comment (asaikarthikeya) [inline org.envirocar.app/res/layout/activity_help_layout.xml:44]: only email works, here. No need of email| web.

**Review by asaikarthikeya — CHANGES_REQUESTED** [review]

File: org.envirocar.app/res/layout/activity_help_layout.xml (hunk 1 above)
Comment (cdhiraj40) [reply org.envirocar.app/res/layout/activity_help_layout.xml:44]: There is a link too, so we gotta add `` web `` too

Comment (asaikarthikeya) [reply org.envirocar.app/res/layout/activity_help_layout.xml:44]: Yes, we do have a web page with that but for feedback we prefer mail.

Comment (cdhiraj40) [reply org.envirocar.app/res/layout/activity_help_layout.xml:44]: Do I have to remove the link cause with only `` email `` the link won't work?
sorry I didn't understand properly

Comment (asaikarthikeya) [reply org.envirocar.app/res/layout/activity_help_layout.xml:44]: Works fine. Merged !

**Pull request merged**